libtransmission/torrent.cc

```diff
@@ -1497,6 +1428,8 @@ tr_stat const* tr_torrentStat(tr_torrent* tor)
     auto seed_ratio_bytes_goal = uint64_t{};
     bool const seed_ratio_applies = tr_torrentGetSeedRatioBytes(tor, &seed_ratio_bytes_left, &seed_ratio_bytes_goal);
 
+    s->eta = TR_ETA_NOT_AVAIL;
+    s->etaIdle = TR_ETA_NOT_AVAIL;
```

Comment:
These lines can go:

https://github.com/transmission/transmission/blob/24068f75d72262e0e585fcb0d168baf49d000e77/libtransmission/torrent.cc#L1450

https://github.com/transmission/transmission/blob/24068f75d72262e0e585fcb0d168baf49d000e77/libtransmission/torrent.cc#L1461

https://github.com/transmission/transmission/blob/24068f75d72262e0e585fcb0d168baf49d000e77/libtransmission/torrent.cc#L1500-L1501

These lines can be condensed:

https://github.com/transmission/transmission/blob/24068f75d72262e0e585fcb0d168baf49d000e77/libtransmission/torrent.cc#L1464-L1469

```cpp
    case TR_STATUS_SEED:
        if (seed_ratio_applies)
```